Classes/Command/FluidTemplatePrecompileCommand.php

```diff
@@ -0,0 +1,161 @@
+            $outputHelper->warning(
+                "\nIt seems like there is an error handler registered."
+                . "\nIn some cases this can prevent the precompiling to run properly."
+                . "\nAn ErrorHandler might raise an exception for a PHP Warning."
+            );
+
+            restore_error_handler();
+        }
+
+        $result = $this->getPrecompilerService()->warmup($extension);
+
+        // General information
+        $templatesCount = 0;
+        foreach ($result as $extensionResult) {
+            $templatesCount += count($extensionResult);
+        }
+
+        $uncompilable = array_column($result, 'uncompilable');
+        $uncompilable = array_sum($uncompilable);
+
+        $outputHelper->section('General');
+        $outputHelper->text(vsprintf('Compiled %d extension(s) with %d template(s) in total.', [count($result), $templatesCount]));
+        $outputHelper->text(vsprintf('%d template(s) could not be compiled.', [$uncompilable]));
+
+        // Additional information
+        if ($output->isVerbose()) {
+            foreach ($result as $extensionKey => $extensionResult) {
+                $templates = $extensionResult['results'];
+
+                if ($onlyFailed) {
+                    $templates = array_filter($templates, function ($template) {
+                        return false == $template[FluidCacheWarmupResult::RESULT_COMPILABLE];
+                    });
+                }
+
+                $templates = array_slice($templates, 0, $limit);
+
+                $templateRows = [];
+                foreach ($templates as $templateName => $templateAttributes) {
+                    $templateRows[] = ['Template', $templateName];
+
+                    foreach ($templateAttributes as $attributeName => $attributeValue) {
+                        $templateRows[] = [
+                            $attributeName,
+                            is_array($attributeValue) ? implode(PHP_EOL, $attributeValue) : static::formatReadable($attributeValue)
+                        ];
+                    }
+
+                    $templateRows[] = new TableSeparator();
+                }
+
+                array_pop($templateRows);
+
+                $outputHelper->section(sprintf('Extension: %s (%d uncompilable of %d)', $extensionKey, $extensionResult['uncompilable'], count($templates)));
+
+                $outputHelper->table(
+                    [],
+                    $templateRows
+                );
+            }
+        }
+
+        if ($disableErrorHandlers) {
+            restore_error_handler();
+        }
+
+        if ($fail && $uncompilable > 0) {
+            throw new \Exception(sprintf('Could not compile %d templates.', $uncompilable), 1518205183);
+        }
+    }
+
+    /**
+     * Formats the given input in a readable format
+     *
+     * @param mixed $data
+     * @return string
+     */
+    protected static function formatReadable($data)
```

Comment:
This method does not need to be static - and there is only a single usage. I'd remove it.